src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommand.java

```diff
@@ -25,27 +24,25 @@
 
 /**
  * Edits an existing Activity in the address book and updates the travel plan/wishlist in the current directory
- * Edits the importance, cost, location, startdate, enddate
+ * An activity contains the field name, location, cost, level of importance and date time
  */
 public class EditActivityCommand extends EditCommand {
     public static final String COMMAND_WORD = "activity";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Edits the travel plan identified by the index number used in the displayed travel planner list.\n"
+            + ": Edits the activity identified by the index number used in the displayed travel planner list.\n"
```

Comment:
Is it travel plan list?

Reply:
I am intending to specify it to "displayed activity list"!